chain/gen/genesis/genesis.go

```diff
@@ -266,6 +263,12 @@ func MakeGenesisBlock(ctx context.Context, bs bstore.Blockstore, sys runtime.Sys
 		BLSAggregate:          nil,
 		BlockSig:              nil,
 		Timestamp:             template.Timestamp,
+		BeaconEntries: []types.BeaconEntry{
+			{
+				Round: 0,
+				Data:  make([]byte, 32),
```

Comment:
We should be getting this from DrandBeacon, right? Wiring it here should be pretty trivial.

Reply:
I think actually we want to use some value we pick for the genesis block in any case, otherwise we can't make the genesis block until the particular time that we want to start the network